app/src/main/java/com/github/se/project/screens/SignInScreen.kt

```diff
@@ -1,29 +1,174 @@
+                                  },
+                              style = Typography.bodyLarge, // Apply the text style from the theme
+                              textAlign = TextAlign.Center,
+                              color =
+                                  MaterialTheme.colorScheme.onBackground, // Use color from theme
+                              modifier = Modifier.fillMaxWidth().padding(horizontal = 42.dp))
+                        }
+                  }
+
+              Spacer(modifier = Modifier.height(32.dp))
+
+              // Pagination Dots
+              Row(horizontalArrangement = Arrangement.Center, modifier = Modifier.fillMaxWidth()) {
+                PaginationDot(isActive = pagerState.currentPage == 0)
+                Spacer(modifier = Modifier.width(8.dp))
+                PaginationDot(isActive = pagerState.currentPage == 1)
+                Spacer(modifier = Modifier.width(8.dp))
+                PaginationDot(isActive = pagerState.currentPage == 2)
+              }
+
+              Spacer(modifier = Modifier.height(32.dp))
+
+              // Google Sign-in Button
+              Button(
+                  onClick = onSignInClick,
+                  shape = Shapes.medium, // Apply the custom shape from the theme
+                  modifier =
+                      Modifier.fillMaxWidth(0.8f)
+                          .height(48.dp)
+                          .clip(Shapes.medium)
+                          .testTag("loginButton"),
+                  colors =
+                      ButtonColors(
+                          MaterialTheme.colorScheme.secondary,
+                          MaterialTheme.colorScheme.onPrimary,
+                          MaterialTheme.colorScheme.secondary,
+                          MaterialTheme.colorScheme.onSecondary)) {
+                    Image(
+                        painter = painterResource(id = R.drawable.google_logo), // Google logo
+                        contentDescription = null,
+                        modifier = Modifier.size(24.dp))
+                    Spacer(modifier = Modifier.width(8.dp))
+                    Text(
+                        text = "Continue with Google",
+                        style = Typography.labelMedium) // Apply button text style
+              }
+
+              Spacer(modifier = Modifier.height(16.dp))
+
+              // Terms and Conditions
+              Text(
+                  text =
+                      "By clicking 'Continue', you agree to the Terms of Service and Privacy Policy, and consent to the use of cookies related to PocketTutor.",
+                  style = Typography.bodySmall, // Use bodySmall style from the theme
+                  color = MaterialTheme.colorScheme.onBackground, // Use color from theme
+                  modifier = Modifier.padding(horizontal = 32.dp),
+                  textAlign = TextAlign.Center)
+            }
```

Comment:
Do we have Terms and Conditions yet?

Reply:
no but imagine I imagine we will have some